addons/website/static/src/snippets/s_image_gallery/options.js

```diff
@@ -320,6 +322,14 @@ options.registry.gallery = options.Class.extend({
             attrStyle: imageEls.length > 0 ? imageEls[0].style.cssText : '',
         },
         $slideshow = $(qweb.render('website.gallery.slideshow', params));
+        const imgSlideshowEls = $slideshow[0].querySelectorAll("img");
+        for (const index in linkedImageEls) {
```

Comment:
Never use `for...in` to loop through array indexes. I really don't think it is a good practice. Here for instance, any (crappy) code extending the `_getLinkedImages` function to do something like this:
```javascript
_getLinkedImages: function () {
    const els = this._super(...arguments);
    els.onlyContainsImages = els.every(el => el.tagName === "IMG");
    return els;
},
```
would crash as `index` would be equal to `"onlyContainsImages"` at some point in the loop.